zzz_generated/all-clusters-app/zap-generated/af-gen-event.h

```diff
@@ -87,8 +82,7 @@
 
 // EmberEventData structs used to populate the EmberEventData table
 #define EMBER_AF_GENERATED_EVENTS                                                                                                  \
-    { &emberAfIdentifyClusterServerTickCallbackControl1, emberAfIdentifyClusterServerTickCallbackWrapperFunction1 },               \
-        { &emberAfLevelControlClusterServerTickCallbackControl1, emberAfLevelControlClusterServerTickCallbackWrapperFunction1 },   \
+    { &emberAfLevelControlClusterServerTickCallbackControl1, emberAfLevelControlClusterServerTickCallbackWrapperFunction1 },       \
```

Comment:
For my understanding: This change is the result of automatic regeneration triggered by a change to all-clusters-app.zap but the Identify cluster does not actually use any part of the Ember event mechanism -- is this correct?

Reply:
No, I had to edit that part manually because there is no code which generates this file. So I don't know how this file was initially generated. Even the af-gen-event.zapt file does not match the generated output.

Reply:
Could we update af-gen-event to generate these and use it?

Manual generation is an issue - we lost gen-event somehow.
@vivien-apple ?

How are events used in general? Do them map to  matter events or will they?

Reply:
Well, that was what I was initially asking on slack. But didn't got a clear response. According to the slack discussion. 

> Sergei Lissianoi:
To restart the discussion of replacing Ember/Silabs event mechanism with chip::DeviceLayer::SystemLayer.StartTimer : after looking through the code I think it can and should be done.

And I do thing that we should get rid of the af-gen-event.h because it will just increase the code size with generated look up tables.